### PR TITLE
Add FP8 MFMA/GG support and several fix

### PR DESCRIPTION
**Pull request by KKyang**
Currently the default bias_type is set to match the yamls' configuration, should change the default type when the yamls are fixed.

### CODE REVIEW

**Review by aazz44ss — APPROVED** [review]

**Comment by KKyang** [comment]
gfx94x local test passed

File: tensilelite/Tensile/KernelWriterAssembly.py [hunk 1]
```diff
@@ -1122,6 +1122,13 @@ def defineAndResources(self, kernel, tPA, tPB, lralwaCode):
       load = self.states.numSgprToLoad
       sgprStart = self.sgprs["SizesFree"]
       moduleLoadAllKernArg = self.argLoader.loadAllKernArg(sgprStart, "KernArgAddress", load, 0)
+      if kernel["ProblemType"]["UseScaleAB"]:
+        sgprOffset = self.argLoader.getOffset()
+        for name in ['A','B']:
```
Comment (jichangjichang) [inline tensilelite/Tensile/KernelWriterAssembly.py:1127]: It seems that   self.states.preloadScaleA  and   self.states.preloadScaleB can replace this kind of checking.

Comment (KKyang) [reply tensilelite/Tensile/KernelWriterAssembly.py]: Fixed

File: tensilelite/Tensile/KernelWriter.py [hunk 2]
```diff
@@ -3449,6 +3451,19 @@ def readWriteVectors(mat, vw, kernel):
     # for conditionals
     self.states.lastPostLoopSgpr = self.sgprPool.size()
 
+    self.states.preloadScaleA = False
```
Comment (jichangjichang) [inline tensilelite/Tensile/KernelWriter.py:3454]: do we need this ?
If so, it also need to init preloadScaleB

Comment (KKyang) [reply tensilelite/Tensile/KernelWriter.py:3454]: fixed

**Review by jichangjichang — APPROVED** [review]
LGTM

**Pull request merged**